x/slashing/handler.go

```diff
@@ -1,9 +1,31 @@
 package slashing
 
 import (
+	"math/big"
+	"strings"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+var (
+	SideChainIdBsc    = big.NewInt(56)
+	SideChainIdChapel = big.NewInt(97)
+	SideChainIdRialto = big.NewInt(714)
+)
+
+func SideChainIdFromText(str string) (*big.Int, error) {
+	switch strings.ToLower(str) {
+	case "bsc":
+		return SideChainIdBsc, nil
+	case "chapel":
+		return SideChainIdChapel, nil
+	case "rialto":
+		return SideChainIdRialto, nil
+	default:
+		return big.NewInt(-1), ErrInvalidSideChainId(DefaultCodespace)
```

Comment:
return `big.NewInt(-1)` is unnecessary, we can `return nil, ErrInvalidSideChainId(DefaultCodespace)`